lib/shopify_api/resources/checkout.rb

```diff
@@ -1,4 +1,31 @@
+require 'shopify_api/resources/shipping_rate'
+
 module ShopifyAPI
   class Checkout < Base
+    has_many :shipping_rates, class_name: ShopifyAPI::ShippingRate
```

Comment:
@JakeCataford this seems to be unsupported by earlier versions of active resource :/